django/db/backends/mysql/features.py

```diff
@@ -70,6 +70,14 @@ def can_introspect_foreign_keys(self):
         "Confirm support for introspected foreign keys"
         return self._mysql_storage_engine != 'MyISAM'
 
+    @cached_property
+    def can_return_columns_from_insert(self):
+        return self.connection.mysql_is_mariadb and self.connection.mysql_version >= (10, 5, 0)
+
+    @cached_property
+    def can_return_rows_from_bulk_insert(self):
+        return self.connection.mysql_is_mariadb and self.connection.mysql_version >= (10, 5, 0)
```

Comment:
We can reuse `can_return_columns_from_insert` here:
```python
can_return_rows_from_bulk_insert = property(operator.attrgetter('can_return_columns_from_insert'))
```

Reply:
Yup